src/pages/PostsList/components/PostPreview.jsx

```diff
@@ -0,0 +1,42 @@
+import { LikeOutlined } from '@ant-design/icons'
+import { Button, List, Space, Typography } from 'antd'
+import React from 'react'
+
+const { Paragraph } = Typography
+
+const IconText = ({ icon, text }) => (
+  <Space>
+    {React.createElement(icon)}
+    {text}
+  </Space>
+)
+
+const PostPreview = ({ post }) => {
+  return (
```

Comment:
If you only return something inside component, you can use this pattern:
```suggestion
const PostPreview = ({ post }) => <List>... </List>
```
So without curly brackets and `return` statement